app/views/dashboard/assessmentDetails.scala.html

```diff
@@ -93,7 +93,7 @@
                                     }
                                 }
                             </td>
-                            <td>
+                            <td class="govuk-table__cell">
                                 @assessment.rateableValue match {
                                     case Some(rv) => {
                                         <a id="viewAssessmentRadio-@assessment.assessmentRef" href="@url">@messages("assessments.historic.select")</a>
```

Comment:
this needs to have `govuk-link` css class

Reply:
"govuk-link" should go on the anchor tag, not the td

Reply:
yes, i'm commenting on line 99. The a tag was left alone and it needed updating